packages/esm-patient-notes-app/src/notes/visit-notes.resource.ts

```diff
@@ -63,6 +64,45 @@ export function useVisitNotes(patientUuid: string): UseVisitNotes {
   };
 }
 
+export function useInfiniteVisits(patientUuid: string) {
```

Comment:
@vasharma05, @ibacher, @brandones  should this hook move to Common Lib instead? It's a copy of what exists in the Patient Chart [here](https://github.com/openmrs/openmrs-esm-patient-chart/blob/main/packages/esm-patient-chart-app/src/visit/visits-widget/visit.resource.tsx#L13).

Reply:
Seems reasonable. It has a simple API and what it does is fairly complex; and it will be good to ensure that the cache gets hit correctly.